modules/openxr/openxr_api_extension.h

```diff
@@ -70,7 +72,21 @@ class OpenXRAPIExtension : public RefCounted {
 	int64_t get_next_frame_time();
 	bool can_render();
 
+	void register_composition_layer_provider(OpenXRExtensionWrapperExtension *p_extension);
+	void unregister_composition_layer_provider(OpenXRExtensionWrapperExtension *p_extension);
+
+	enum OpenXRAlphaBlendModeSupport {
+		OPENXR_ALPHA_BLEND_MODE_SUPPORT_NONE = 0,
+		OPENXR_ALPHA_BLEND_MODE_SUPPORT_REAL = 1,
+		OPENXR_ALPHA_BLEND_MODE_SUPPORT_EMULATING = 2,
+	};
```

Comment:
Out of curiosity, is it possible to reuse the same enum declaration here and in `openxr_api.h` instead of redeclaring it in both places?

Reply:
I don't think so. I tried using both a `typedef` and `using` to pull the `enum` in from `OpenXRAPI` but the binding layer wouldn't recognize it correctly as being part of the class. Maybe someone else knows a way to do it?